### PR TITLE
Sandbox api

### PR DESCRIPTION
**Pull request by Ekrekr**

### CODE REVIEW

File: sandbox/executor.cc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    ssize_t stderrRLen = read(errFd, stderrBuf, sizeof(stderrBuf));
+    printf("stdout: '%s'\n", std::string(stdoutBuf, stdoutRLen).c_str());
+    printf("stderr: '%s'\n", std::string(stderrBuf, stderrRLen).c_str());
+    if (stdoutRLen < 1) {
+      break;
+    }
+  }
+}
+
+std::string GetDataDependencyFilePath(absl::string_view relative_path) {
```
Comment (cblichmann) [inline sandbox/executor.cc:119]: Sandboxed API 19fd11b91e6678db2fcfe69dd4037605730f5317 made `util/runfiles.h` public, so you may want to update the dependency.

Comment (Ekrekr) [reply sandbox/executor.cc:119]: Awesome thanks

**Comment by Ekrekr** [comment]
Debugging command: `bazel run //sandbox:executor --verbose_failures -- --sandbox2_danger_danger_permit_all=true --logtostderr --v=1`

**Comment by Ekrekr** [comment]
Update: Depends on https://github.com/dataform-co/dataform/pull/1179 and https://github.com/dataform-co/dataform-co/pull/9407

**Pull request closed without merging**